test/integration/action_execution.go

```diff
@@ -704,3 +748,9 @@ func getTargetSENodeName(f *framework.TestFramework, pod *corev1.Pod) string {
 func podID(pod *corev1.Pod) string {
 	return fmt.Sprintf("%s/%s", pod.Namespace, pod.Name)
 }
+
+func validateResourceResize(oldResourceQuan, newResourceQuan resource.Quantity) {
```

Comment:
This is not generic.  Can we make it generic by changing the signature to (oldResourceVal, newResourceVal, increment)? And it should cover both cpu and memory in case we want to implement a test case for memory resize.

Reply:
@enlinxu  what do you think of this definition `func validateResourceResize(oldResourceVal, newResourceVal resource.Quantity,increment int64)`?

Reply:
@enlinxu , as Irfan suggested, I remove this function, putting the logic of the validation in the function `waitForResourcesUpdated` instead.